.envrc

```
@@ -0,0 +1,30 @@
+if ! find_up .venv/pyvenv.cfg; then
+    echo "ITS NOT there"
+    python3 -m venv .venv
+    echo "NOW IT IS"
+fi
```

```suggestion
if [ ! -d "$VENV_PATH" ]; then
    echo "info: creating virtual environment under: $VENV_PATH"
    
    python3 -m venv "$VENV_PATH"
fi
```

A few notes:
- Create a variable for the venv path so that you can reference it in the rest of the file.
- Use a regular shell conditional check, look at the `test` man page to learn more about these conditions. Also, check it the venv exists and not some file on the inside as I don't believe `pyvenv.cfg` has always been there.
- Avoid diagnostic messages that are not descriptive. It's okay to include these messages as it let's a user know what's going on, but include more info to tell the user that you're creating a venv and where.